physionet-django/physionet/views.py

```diff
@@ -119,7 +125,7 @@ def database_overview(request):
         projects[i]['policy'] = policy
         projects[i]['projects'] = PublishedProject.objects.filter(
             access_policy=i, resource_type=0, is_latest_version=True
-            ).order_by(Lower('title'))
+        ).order_by(Lower('title'))
```

Comment:
I assume these changes were automatically applied by your code editor! Perhaps leave this for now, but I think it's generally best not to make styling changes like this (even though the style is an improvement) because (1) it makes it more difficult for the reviewer to spot functional code changes (2) it increases chance of a conflict with another pull request). Styling changes could be made in a new PR.

Reply:
thanks, I think these were automatically added by my editor. I will avoid this in the future.